src/main_controller.ts

```diff
@@ -360,6 +368,8 @@ export class MainController implements vscode.Disposable {
     };
 
     private setClientInfo() {
+        // We know version will be present in package.json...
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
         const versionString = this.extContext.extension.packageJSON.version as string;
```

Comment:
Will this throw an error?
```suggestion
        // We know version will be present in package.json...
        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
        const versionString = (this.extContext.extension.packageJSON as any).version as string;
```

Reply:
Yes, the compiler already things `packageJSON` is `any`.